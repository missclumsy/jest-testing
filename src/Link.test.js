import React from 'react';
import renderer from 'react-test-renderer';
import Link from './Link';

describe('<Link />', () => {
  const link = <Link page='lol.com'>Link</Link>

  test('renders correctly', () => {
    const component = renderer.create(link).toJSON();
    expect(component).toMatchSnapshot();
  })
})
