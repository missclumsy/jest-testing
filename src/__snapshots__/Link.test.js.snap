// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> renders correctly 1`] = `
<a
  className="normal"
  href="lol.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Link
</a>
`;
